tdb/vidrl_upload.py

```diff
@@ -71,8 +71,10 @@ def parse_human_serum_references(human_serum_data, subtype):
             # Use the first match of the potential fields
             if matches is not None:
                 year = matches.group(2)
-                # Follow a standard pattern where serum_id is `SH {year}`
-                new_serum_id = f"SH {year}"
+                # Follow a standard pattern where serum_id is `Human pool <year>`
+                # Need "human" in serum_id because this is how we match for human sera in seasonal flu
+                # <https://github.com/nextstrain/seasonal-flu/blob/89f6cfd11481b2c51c50d68822c18d46ed56db51/workflow/snakemake_rules/download_from_fauna.smk#L93>
+                new_serum_id = f"Human pool {year}"
```

Comment:
OH! This actually causes `serum_host` to be set to `human` in fauna because of 

https://github.com/nextstrain/fauna/blob/88a607db53d36fc91482cae2009eefddf9477f97/tdb/upload.py#L382-L383